### PR TITLE
Use markers for conditional dependencies.

### PR DESCRIPTION
**Pull request by Lukasa**
Resolves #252.

/cc @dstufft

### CODE REVIEW

**Comment by dstufft** [comment]
🚢

File: setup.py [hunk 1]
```diff
@@ -114,5 +88,14 @@ def resolve_install_requires():
     },
     extras_require={
         'fast': ['pycohttpparser'],
+        # Fallback to good SSL on bad Python versions.
+        ':python_full_version < "2.7.9" or python_version == "3.3"': [
```
Comment (sigmavirus24) [inline setup.py]: You don't indicate that 3.3 is supported in the classifiers. Why are you accommodating it here then?

Comment (Lukasa) [inline setup.py]: That's an excellent question.

**Comment by Lukasa** [comment]
Cool, this should be fine now.

**Pull request merged**